src/views/Upload.vue

```diff
@@ -199,33 +199,47 @@ export default {
 				}
 			}
 
-			this.$store.dispatch('setLoading', true);
-			this.$store.dispatch('AlertStore/dismissAlert');
-
-			this.$axios
-				.post(
-					'/api/submissions/',
-					isFileUpload ? formData : requestBody,
-					{
-						headers
+			this.$store.dispatch('ModalStore/setModal', {
+				type: 'html',
+				htmlTemplate: {
+					header: this.$i18n.t('emailPolicyTitle'),
```

Comment:
Please, use string interpolation.

https://dmitripavlutin.com/string-interpolation-in-javascript/#:~:text=String%20interpolation%20is%20replacing%20placeholders,%24%7Bexpression%7D%20as%20a%20placeholder.


e.g.: 
```javascript
header: `<h1 class="text-2xl font-bold">${ this.$i18n.t('emailPolicyTitle') }</h1>`,
```